### ISSUE TITLE
Problem on regression of dimension (Formula 3)

### ISSUE THREAD

**Issue by maudzung** [Banconxuan/RTM3D#5]
Hi @Banconxuan 

Thanks for the great paper. I'm really interested in your ideas in the paper.
I have read the paper and tried to re-produce the paper, but I have some questions, I hope you can answer them.

1. In the formula **(3)**, the dimensions _(h, w, l)_ are normalized first, then we compute the MSE loss. But what happens if the normalized dimension is less than 0? In this case, the result of `log` operator will be **NaN**. So I would like you to confirm the formula.
<img width="178" alt="dim_nor" src="https://user-images.githubusercontent.com/15675550/89732525-6dc21780-da8a-11ea-9848-f31a2a49d3df.png">

2. Is it `argmin` instead of `argmax` in the formular **(7)**?
<img width="500" alt="3d_box" src="https://user-images.githubusercontent.com/15675550/89732649-1e301b80-da8b-11ea-852a-26bbf9c495a0.png">

3. To generate heatmaps, we need sigma values. I have followed the paper and found the min/max area of 2D boxes (in the original size) in the training data as bellow:
```
max_sigma = 19
min_sigma = 3
Amax = 129682
Amin = 13
```
<img width="183" alt="sigma" src="https://user-images.githubusercontent.com/15675550/89732708-98f93680-da8b-11ea-803a-47be652b5843.png">

Are the `Amin`, `Amax` similar in your implementation?

Thank you once again!
